src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java

```diff
@@ -317,7 +317,7 @@ public void captures_correctly_when_captor_used_on_pure_vararg_method() throws E
     }
 
     @Test
-    public void should_capture_null_vararg() {
+    public void should_capture_null_single_argument_in_vararg_method() {
```

Comment:
This one is not explicit about casting.

Sorry I'm picky and on a mobile phone ;)